test/unit/Cardano/NetworkLayer/HttpBridgeSpec.hs

```diff
@@ -1,5 +1,7 @@
 module Cardano.NetworkLayer.HttpBridgeSpec
     ( spec
+    , mockNetworkLayer
+    , noLog
```

Comment:
This feels wrong. Tests shouldn't depend on each others. I'd rather have a bit of duplication than shared code like this between tests. The rational being that the `mockNetworkLayer` was designed for the `HttpBridgeSpec` (and it goes the same for Orphan instances). Having each `***Spec` file independent allows for test to be very easy to follow and maintain in the long-run. Making changes to one won't impact the others. 

Especially because here, you don't need the "power" granted by the `mockNetworkLayer` from the HttpBridge. You could have a dummy network layer that would yield empty list of blocks working just fine for your use-case.

Reply:
corrected